README.md

```diff
@@ -39,10 +39,6 @@ built from source.
 Some operating systems already provide binaries for Alacritty, for everyone else there are
 instructions to compile Alacritty from source.
```

Comment:
I think having a link at the top is a great idea. However I question if both is really necessary.

Maybe just adding the link to the top in an explicit manner would suffice and then the manual installation section could be completely removed?

Or do you think it's best to have both present, just in case people are explicitly searching for manual installation instructions?

Reply:
I don't have a strong opinion on this. The link at the top, plus the fact there is an `INSTALL.md` at the root of the repo is pretty visible.

Your call ;)

Reply:
Thanks for the feedback. I'll remove the dedicated section and put the link at the top, to reduce maintenance cost in the future.